Source/gui/SNTBinaryMessageWindowView.swift

```diff
@@ -156,6 +177,7 @@ struct SNTBinaryMessageEventView: View {
   let window: NSWindow?
```

Comment:
Turns out I didn't need this, please remove

Reply:
Done